opencolorio_config_aces/config/studio/generate/resources/OpenColorIO-Config-ACES CG and Studio Transforms - v1.0.0-rc1 - Studio Config - Mapping.csv

```diff
@@ -4,17 +4,18 @@ Ordering,Legacy Name,ACEStransformID,CLFtransformID,Interface,BuiltinTransform S
 100,ACES - ACEScg,urn:ampas:aces:transformId:v1.5:ACEScsc.Academy.ACEScg_to_ACES.a1.0.3,,ColorSpace,ACEScg_to_ACES2065-1,"acescg,lin_ap1",scene-linear,ACES
 200,Input - ADX - ADX10,urn:ampas:aces:transformId:v1.5:ACEScsc.Academy.ADX10_to_ACES.a1.0.3,,ColorSpace,ADX10_to_ACES2065-1,adx10,log,ACES
 200,Input - ADX - ADX16,urn:ampas:aces:transformId:v1.5:ACEScsc.Academy.ADX16_to_ACES.a1.0.3,,ColorSpace,ADX16_to_ACES2065-1,adx16,log,ACES
-203,Input - ARRI - Curve - V3 LogC (EI800),,,NamedTransform,,crv_logc3ei800,log,ARRI
-203,Input - ARRI - Linear - ALEXA Wide Gamut,,,ColorSpace,,lin_awg,scene-linear,ARRI
-203,Input - ARRI - V3 LogC (EI800) - Wide Gamut,urn:ampas:aces:transformId:v1.5:IDT.ARRI.Alexa-v3-logC-EI800.a1.v2,,ColorSpace,,logc3ei800_awg,log,ARRI
-204,Input - ARRI - Curve - LogC4,,,NamedTransform,,crv_logc4,log,
-204,Input - ARRI - Linear - ARRI Wide Gamut 4,,,ColorSpace,,lin_awg4,scene-linear,
-204,Input - ARRI LogC4,,,ColorSpace,,logc4_awg4,log,
+203,Input - ARRI - Curve - V3 LogC (EI800),,urn:aswf:ocio:transformId:1.0:ARRI:Input:ARRI_LogC3_Curve_EI800_to_Linear:1.0,NamedTransform,,crv_logc3ei800,log,ARRI
+203,Input - ARRI - Linear - ALEXA Wide Gamut,,urn:aswf:ocio:transformId:1.0:ARRI:Input:Linear_ARRI_Wide_Gamut_3_to_ACES2065-1:1.0,ColorSpace,,lin_awg,scene-linear,ARRI
+203,Input - ARRI - V3 LogC (EI800) - Wide Gamut,urn:ampas:aces:transformId:v1.5:IDT.ARRI.Alexa-v3-logC-EI800.a1.v2,urn:aswf:ocio:transformId:1.0:ARRI:Input:ARRI_LogC3_EI800_to_ACES2065-1:1.0,ColorSpace,,logc3ei800_awg,log,ARRI
+204,Input - ARRI - Curve - LogC4,,urn:aswf:ocio:transformId:1.0:ARRI:Input:ARRI_LogC4_Curve_to_Linear:1.0,NamedTransform,,crv_logc4,log,
+204,Input - ARRI - Linear - ARRI Wide Gamut 4,,urn:aswf:ocio:transformId:1.0:ARRI:Input:Linear_ARRI_Wide_Gamut_4_to_ACES2065-1:1.0,ColorSpace,,lin_awg4,scene-linear,
+204,Input - ARRI LogC4,,urn:aswf:ocio:transformId:1.0:ARRI:Input:ARRI_LogC4_to_ACES2065-1:1.0,ColorSpace,,logc4_awg4,log,
 210,Input - Blackmagic - Curve - DaVinci Intermediate,,urn:aswf:ocio:transformId:1.0:BlackmagicDesign:Input:DaVinci_Intermediate_Log-Curve_to_Linear:1.0,NamedTransform,,crv_davinci_intermediate,log,Blackmagic
 210,Input - Blackmagic - DaVinci Intermediate - DaVinci Wide Gamut,,urn:aswf:ocio:transformId:1.0:BlackmagicDesign:Input:DaVinci_Intermediate_WideGamut_to_ACES2065-1:1.0,ColorSpace,,davinci_intermediate_dwg,log,Blackmagic
 210,Input - Blackmagic - Linear - DaVinci Wide Gamut,,urn:aswf:ocio:transformId:1.0:BlackmagicDesign:Input:Linear_DaVinci_WideGamut_to_ACES2065-1:1.0,ColorSpace,,lin_dwg,scene-linear,Blackmagic
+210,Input - Canon - Canon-Log3 - Cinema Gamut Daylight,,urn:aswf:ocio:transformId:1.0:Canon:Input:CanonLog3_CinemaGamut-D55_to_ACES2065-1:1.0,ColorSpace,CANON_CLOG3-CGAMUT_to_ACES2065-1,clog3_cgamut,log,Canon

```

Comment:
The alias intended to match the compact Utility/Aliases name of the previous config is incorrect for several of the spaces.  For example, this entry from the spreadsheet says "clog3_cgamut".  The aliases in the Studio config are: 
`[canonlog3_cinemagamut_d55, Input - Canon - Canon-Log3 - Cinema Gamut Daylight, clog3_cgamut]
`

So, the spreadsheet value makes it into the config, but unfortunately it is the wrong value.  The value from the previous config is "canonlog3_cgamutday".  And indeed, the spreadsheet has that in the "Legacy Aliases" column, but it doesn't get used by the generator.

This is not the only example and I will write a script to print out all of them tomorrow.  But my question is, where did the "clog3_cgamut" come from and why is it being used rather than the "Legacy Aliases" value?

Reply:
The `Legacy Aliases` column is not used currently, the `Aliases` column is as we discussed, @michdolan did update them, the other aliases are automatically generated: https://github.com/AcademySoftwareFoundation/OpenColorIO-Config-ACES/blob/ff3f524884258196dde9a720fcd43236650090b1/opencolorio_config_aces/config/generation/beautifiers.py#L410

Reply:
I wrote a script that goes through each color space in the Studio config generated from this branch and then, for each of its aliases, tries to find a color space with that name in the legacy ACES 1.2 config.  Here are the results:

sRGB - Display
    "srgb_display" -- Not found
 
Rec.1886 / Rec.709 Video - Display
    "rec1886_rec709_video_display" -- Not found
 
Rec.1886 / Rec.2020 Video - Display
    "rec1886_rec2020_video_display" -- Not found
 
Rec.2100-HLG - Display
    "rec2100_hlg_display" -- Not found
 
Rec.2100-PQ - Display
    "rec2100_pq_display" -- Not found
 
ST2084-P3-D65 - Display
    "st2084_p3_d65_display" -- Not found
 
P3-D60 - Display
    "p3_d60_display" -- Not found
 
P3-D65 - Display
    "p3_d65_display" -- Not found
 
P3-DCI - Display
    "p3_dci_display" -- Not found
 
ACES2065-1
    "aces2065_1" -- Not found
    "lin_ap0" -- Match!
 
Raw
 
ACEScc
    "acescc_ap1" -- Match!
 
ACEScct
    "acescct_ap1" -- Match!
 
ACEScg
    "lin_ap1" -- Match!
 
ADX10
 
ADX16
 
Linear ARRI Wide Gamut 3
    "lin_arri_wide_gamut_3" -- Not found
    "Input - ARRI - Linear - ALEXA Wide Gamut" -- Match!
    "lin_awg" -- Not found
 
ARRI LogC3 EI800
    "arri_logc3_ei800" -- Not found
    "Input - ARRI - V3 LogC (EI800) - Wide Gamut" -- Match!
    "logc3ei800_awg" -- Not found
 
Linear ARRI Wide Gamut 4
    "lin_arri_wide_gamut_4" -- Not found
    "Input - ARRI - Linear - ARRI Wide Gamut 4" -- Not found
    "lin_awg4" -- Not found
 
ARRI LogC4
    "arri_logc4" -- Not found
    "Input - ARRI LogC4" -- Not found
    "logc4_awg4" -- Not found
 
DaVinci Intermediate WideGamut
    "davinci_intermediate_widegamut" -- Not found
    "Input - Blackmagic - DaVinci Intermediate - DaVinci Wide Gamut" -- Not found
    "davinci_intermediate_dwg" -- Not found
 
Linear DaVinci WideGamut
    "lin_davinci_widegamut" -- Not found
    "Input - Blackmagic - Linear - DaVinci Wide Gamut" -- Not found
    "lin_dwg" -- Not found
 
CanonLog3 CinemaGamut D55
    "canonlog3_cinemagamut_d55" -- Not found
    "Input - Canon - Canon-Log3 - Cinema Gamut Daylight" -- Match!
    "clog3_cgamut" -- Not found
 
Linear CinemaGamut D55
    "lin_cinemagamut_d55" -- Not found
    "Input - Canon - Linear - Canon Cinema Gamut Daylight" -- Match!
    "lin_cgamut" -- Not found
 
Linear V-Gamut
    "lin_vgamut" -- Match!
    "Input - Panasonic - Linear - V-Gamut" -- Match!
 
V-Log V-Gamut
    "vlog_vgamut" -- Match!
    "Input - Panasonic - V-Log - V-Gamut" -- Match!
 
Linear REDWideGamutRGB
    "lin_redwidegamutrgb" -- Not found
    "Input - RED - Linear - REDWideGamutRGB" -- Match!
    "lin_rwg" -- Match!
 
Log3G10 REDWideGamutRGB
    "log3g10_redwidegamutrgb" -- Not found
    "Input - RED - REDLog3G10 - REDWideGamutRGB" -- Match!
    "log3g10_rwg" -- Not found
 
Linear S-Gamut3
    "lin_sgamut3" -- Match!
    "Input - Sony - Linear - S-Gamut3" -- Match!
 
Linear S-Gamut3.Cine
    "lin_sgamut3cine" -- Match!
    "Input - Sony - Linear - S-Gamut3.Cine" -- Match!
 
Linear Venice S-Gamut3
    "lin_venice_sgamut3" -- Match!
    "Input - Sony - Linear - Venice S-Gamut3" -- Match!
 
Linear Venice S-Gamut3.Cine
    "lin_venice_sgamut3cine" -- Match!
    "Input - Sony - Linear - Venice S-Gamut3.Cine" -- Match!
 
S-Log3 S-Gamut3
    "slog3_sgamut3" -- Match!
    "Input - Sony - S-Log3 - S-Gamut3" -- Match!
 
S-Log3 S-Gamut3.Cine
    "slog3_sgamut3cine" -- Not found
    "Input - Sony - S-Log3 - S-Gamut3.Cine" -- Match!
 
S-Log3 Venice S-Gamut3
    "slog3_venice_sgamut3" -- Match!
    "Input - Sony - S-Log3 - Venice S-Gamut3" -- Match!
 
S-Log3 Venice S-Gamut3.Cine
    "slog3_venice_sgamut3cine" -- Not found
    "Input - Sony - S-Log3 - Venice S-Gamut3.Cine" -- Match!
 
Camera Rec.709
    "camera_rec709" -- Not found
    "Utility - Rec.709 - Camera" -- Match!
    "rec709" -- Not found
 
Linear P3-D65
    "lin_p3_d65" -- Not found
    "Utility - Linear - P3-D65" -- Match!
    "lin_p3d65" -- Match!
 
Linear Rec.2020
    "lin_rec2020" -- Match!
    "Utility - Linear - Rec.2020" -- Match!
 
Linear Rec.709
    "lin_rec709" -- Match!
    "Utility - Linear - Rec.709" -- Match!
    "lin_srgb" -- Match!
    "Linear sRGB" -- Not found
 
Gamma 1.8 Rec.709 - Texture
    "g18_rec709" -- Match!
    "Utility - Gamma 1.8 - Rec.709 - Texture" -- Match!
 
Gamma 2.2 AP1 - Texture
    "g22_ap1" -- Not found
    "Utility - Gamma 2.2 - AP1 - Texture" -- Not found
 
Gamma 2.2 Rec.709 - Texture
    "g22_rec709" -- Match!
    "Utility - Gamma 2.2 - Rec.709 - Texture" -- Match!
 
Gamma 2.4 Rec.709 - Texture
    "g24_rec709" -- Not found
    "Utility - Gamma 2.4 - Rec.709 - Texture" -- Not found
 
sRGB - Texture
    "srgb" -- Not found
    "Utility - sRGB - Texture" -- Match!
    "Input - Generic - sRGB - Texture" -- Match!

Reply:
Hey @doug-walker,

The aliases are taken from the highlighted column here (the unused columns are greyed out):

![image](https://user-images.githubusercontent.com/99779/193508633-91fe240a-7926-4a85-ba93-83f68f8c45eb.png)

So, we should probably merge them.